test/interpreter/function-now.spec.ts

```diff
@@ -9,17 +9,62 @@ function sleepFor( sleepDuration: number ){
 }
 
 describe('Interpreter - function NOW', () => {
+  let originalNow: () => number
+
+  beforeEach(() => {
+    originalNow = Date.now
+    let cnt = 20
+    Date.now = () => {
+      cnt += 1
+      return Date.parse(`1985-08-16T03:45:${cnt}`)
+    }
```

Comment:
A more elegant way will be and it's clear its a mock:

```suggestion
    dateMock = jest.fn().mockReturnValue(new Date('1985-08-16T03:45:30Z').getTime())
    Date.now = dateMock
```

I will get to NOW in the next comment

Reply:
will it work for the multitude of testers that we have? (`jest`, `jasmine`, `karma`)?

Reply:
true, it will not, I forgot about jasmine

Reply:
I don't think we should fight for some hybrid jest/jasmine mock tests and just go with what we have. But of course it is up to you